hr_timesheet_no_closed_project_task/README.rst

```diff
@@ -0,0 +1,53 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License AGPL-3
+
+===================================
+HR Timesheet No Closed Project-Task
+===================================
+
+This module prevents to select closed project or task on timesheet line.
+
```

Comment:
Please add some more info as @sbidoul said: "we want a project manager to be able to close a project to signal that work is done and users cannot timesheet on it anymore, while leaving the contract open for the account manager".
This will clarify the usage a lot